### PR TITLE
Expand the DB code to handle more data

### PR DESCRIPTION
**Pull request by wibblymat**
R: @beaufortfrancois

Thanks!

### CODE REVIEW

File: src/capture-view.ts [hunk 1]
```diff
@@ -82,23 +83,21 @@ export default class CaptureView extends View {
...
-        } else {
-          // TODO: something?
-        }
-      }, 'image/jpg');
+      canvas.toBlob((blob: Blob) => this.storeResult(blob), 'image/jpg');
     }
   }
+
+  private storeResult(blob: Blob) {
+    const record = new ImageRecord();
```
Comment (beaufortfrancois) [inline src/capture-view.ts]: How about simply passing the original blob to the constructor?
```ts
const record = new ImageRecord(blob);
```

**Review by beaufortfrancois — APPROVED** [review]
LGTM but nit

**Comment by beaufortfrancois** [comment]
See https://github.com/GoogleChrome/snapshot/issues/8 as well.

File: src/capture-view.ts (hunk 1 above)
Comment (wibblymat) [reply src/capture-view.ts]: Done

**Pull request merged**